packages/translate/lib/startup.ts

```diff
@@ -0,0 +1,118 @@
+			};
+		case NOT_STARTED_REASON.NO_ACTIVE_PLAN:
+			return {
+				title: t['startupError.noActivePlan'](),
+				description: t['startupError.noActivePlan.desc'](),
+			};
+		case NOT_STARTED_REASON.CANDYBOX_OFFLINE:
+			return {
+				title: t['startupError.candyBoxOffline'](),
+				description: t['startupError.candyBoxOffline.desc'](),
+			};
+		case NOT_STARTED_REASON.SUITEST_DRIVE_OFFLINE:
+			return {
+				title: t['startupError.suitestDriveOffline'](),
+				description: t['startupError.suitestDriveOffline.desc'](),
+			};
+		case NOT_STARTED_REASON.BOOTING:
+			return {
+				title: t['startupError.runningBootSequence'](),
+				description: t['startupError.runningBootSequence.desc'](),
+			};
+		case NOT_STARTED_REASON.DEVICE_IN_USE:
+			return {
+				title: t['startupError.deviceInUse'](),
+				description: t['startupError.deviceInUse.desc'](),
+			};
+		case NOT_STARTED_REASON.DEVICE_DISABLED:
+			return {
+				title: t['startupError.deviceDisabled'](),
+				description: t['startupError.deviceDisabled.desc'](),
+			};
+		case NOT_STARTED_REASON.DEVICE_DELETED:
+			return {
+				title: t['startupError.deviceDeleted'](),
+				description: t['startupError.deviceDeleted.desc'](),
+			};
+		case NOT_STARTED_REASON.INTERNAL_ERROR:
+			return {
+				title: t['startupError.internalError'](),
+				description: t['startupError.internalError.desc'](),
+			};
+		case NOT_STARTED_REASON.PLATFORM_ERROR_UNDEFINED:
+			return {
+				title: t['startupError.notDefinedPlatform'](),
+				description: t['startupError.notDefinedPlatform.desc'](),
+			};
+		case NOT_STARTED_REASON.PLATFORM_ERROR_LG_WEBOS:
+			return {
+				title: t['startupError.lgWebosPlatformError'](),
+				description: t['startupError.lgWebosPlatformError.desc'](),
+			};
+		case NOT_STARTED_REASON.PLATFORM_ERROR_XBOX:
+			return {
+				title: t['startupError.xboxPlatformError'](),
+				description: t['startupError.xboxPlatformError.desc'](),
+			};
+		case NOT_STARTED_REASON.PLATFORM_ERROR_ANDROID:
+			return {
+				title: t['startupError.androidPlatformError'](),
+				description: t['startupError.androidPlatformError.desc'](),
+			};
+		case NOT_STARTED_REASON.PLATFORM_ERROR_APPLE_TV:
+			return {
+				title: t['startupError.applePlatformError'](),
+				description: t['startupError.applePlatformError.desc'](),
+			};
+		case NOT_STARTED_REASON.PLATFORM_ERROR_ROKU:
+			return {
+				title: t['startupError.rokuPlatformError'](),
+				description: t['startupError.rokuPlatformError.desc'](),
+			};
+		case NOT_STARTED_REASON.EXCEEDED_PLAN_LIMITS:
+			return {
+				title: t['startupError.planLimitExceeded'](),
+				description: t['startupError.planLimitExceeded.desc'](),
+			};
+		case NOT_STARTED_REASON.PLATFORM_ERROR_PLAYSTATION:
+			return {
+				title: t['startupError.playstationPlatformError'](),
+				description: t['startupError.playstationPlatformError.desc'](),
+			};
+		default:
+			const _code: never = code;
+			console.warn(_code, 'startup error code does not exist');
```

Comment:
Think again if we need this console.warn

Reply:
I would drop it actually